src/components/CommonComponents/CardContainer/index.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Typography, Empty, Image } from 'antd';
+import PropTypes from 'prop-types';
+
+import WorkspaceCard from '../WorkspaceCard';
+import arrow from '../../../assets/icons/arrow.svg';
+import './style.css';
+
+const { Title, Text } = Typography;
+
+const CardContainer = ({ title, searchText, data, seeMoreLink, size }) => (
+  <div>
+    {searchText ? (
```

Comment:
Good job!
When see more is clicked you should display a page with a suitable title ,so add the following line 
```
 {searchText || searchText === 'top-new' ? (
```